websites.py

```diff
@@ -0,0 +1,10 @@
+patterns = {
+	'pinterest': r'^[a-zA-Z0-9_]{3,15}$',
+	'gitlab': r'^[a-zA-Z0-9_\-.]+$',
+	'github': r'^[a-zA-Z0-9-]{1,39}$',
+	'tumblr': r'^[a-zA-Z0-9-]{1,32}$',
+	'behance': r'^[a-zA-Z0-9_\-]{1,30}$',
+	'instagram': r'^[a-zA-Z0-9_.]+$',
+	'twitter': r'^[a-zA-Z0-9_]{5,15}$',
+	'soundcloud': r'^[a-zA-Z0-9_\-]+$'
```

Comment:
per instagram

Reply:
Instagram doesn't let you input more than ``30`` chars in the username field.